src/remotebuild/lib/server.ts

```diff
@@ -51,6 +53,13 @@ class Server {
         var app = express();
         app.use(expressLogger("dev"));
         app.use(errorhandler());
+        if (conf.secure) {
+            app.use(helmet.hsts({
+                maxAge: 1000 * 60 * 60 * 24 * 365, // one year in milliseconds
```

Comment:
What do you think of using a small library for these kind of things like https://www.npmjs.com/package/milliseconds ?